src/components/shared/svg/nora-logo/nora-logo.jsx

```diff
@@ -1,11 +1,10 @@
 import Image from 'next/image';
 import ncLogo from 'images/NC.png';
-import styles from './nora-logo.module.scss';
 
 export default function NoraLogo() {
   return (
-    <div className={styles.container}>
+    <>
       <Image src={ncLogo} alt='main logo' />
```

Comment:
I appreciate this though, but this doesn't need to a component in my opinion, its just one line of code. 😂

If you prefer to leave it as a component that's cool, but its also not an svg so i would relocate it to another folder.